pages/about.html

```diff
@@ -0,0 +1,101 @@
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+        </ul></nav>
+        <nav class="nav-bar">
+            <a class="d-logo" href="../index.html"><span class="logo-span">Y-</span>DEV</a>
+            <a href="#"><img class="humberger" src="/assets/menu.svg" alt="humberger"></a>
+            <ul class="d-nav">
+                <li><a href="../index.html">Home</a></li>
+                <li><a class="nav-links" href="#">Program</a></li>
+                <li><a class="nav-links" href="#">Join</a></li>
+                <li><a class="nav-links" href="#">Sponsor</a></li>
+                <li><a class="nav-links" href="#">News</a></li>
+            </ul>
+            <button class="d-btn">cc campaign</button>
+        </nav>
+        <ul class="mobile-nav">
+            <li><a href="../index.html">Home</a></li>
+             <li><a href="#program">Program</a></li>
+             <li><a href="#join">Join</a></li>
+             <li><a href="#sponsor">Sponsor</a></li>
+             <li><a href="#news">News</a></li>
+     </ul>
+     <div class="times">&times</div>
+    </header>
+    <section class="about-hero">
+        <p class="p-1">"Hello! Dev World"</p>
+        <h1 class="h-1">YOUNG DEV'S GLOBAL SUMMIT UGANDA</h1>
+        <p class="p-2">The Young Dev Summit brings together the community of  young,expert devs.
+            this year's summit is aimed to spreading the love of tech to the young generation
+            With the help of our partners
+         </p>
+         <p class="p-3">
+            Please contact us per Email for further questions about Young Dev's Global Summit
+         </p>
+         <a class="p-4" href="#">haylederrick14@gmail.com</a>
+    </section>
+    <section class="about-logo">
+        <div class="logo-header">
+            <h2>Young Dev's Global Summit Uganda Logo</h2>
+            <div class="line"></div>
+            <p class="logo-paragraph">The logo of young dev's global summit was decided through the logo competition</p>
+        </div>
+        
+        <a class="logo-image" href="#"><span class="img-span">Y-</span>Dev</a>
+    </section>
+    <section class="past-summits">
+        <div class="past-header">
+            <h2>See the past Global Summits</h2>
+            <div class="line"></div>
+            <p>Take a look at the past two Global summits that took place in kampala, uganda</p>
+        </div>
+        <div class="past-container">
+            <div class="card-1">
+                <h3>2013</h3>
+                <p>yd Global Summit in kampala,Uganda</p>
+            </div>
+            <div class="card-2">
+                <h3>2011</h3>
+                <p>Yd Global Summit in Nairobi,Kenya</p>
+            </div>
+        </div>
+    </section>
+    <section class="partners">
+        <div class="partner-header">
+            <h2>partners</h2>
```

Comment:
- [ ] Kindly make this title uppercased, colored with white instead of black, this will increase contrast ratio and make it more readable.
![image](https://user-images.githubusercontent.com/91486791/234962018-b7e1cbb1-d922-47e3-a197-2705ea7fa573.png)
![image](https://user-images.githubusercontent.com/91486791/234962058-ed2ae822-acf5-46fb-85c6-c7aff6f27f9d.png)